<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>307</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OCT-Tools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>761</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="menuVertical" stretch="">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>871</width>
      <height>761</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="contenido" stretch="">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>313</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <widget class="QMenu" name="menuExportar">
     <property name="title">
      <string>Exportar</string>
     </property>
     <addaction name="actionImagen"/>
    </widget>
    <addaction name="actionNuevo"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="separator"/>
    <addaction name="menuExportar"/>
    <addaction name="actionImprimir"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionRehacer"/>
    <addaction name="actionDeshacer"/>
    <addaction name="separator"/>
    <addaction name="actionCortar"/>
    <addaction name="actionCopiar"/>
    <addaction name="actionPegar"/>
   </widget>
   <widget class="QMenu" name="menuOCT">
    <property name="title">
     <string>OCT</string>
    </property>
    <widget class="QMenu" name="menuSegmentar">
     <property name="title">
      <string>Segmentar</string>
     </property>
     <addaction name="actionAlgoritmo_1"/>
     <addaction name="actionAlgoritmo_2"/>
     <addaction name="actionAlgoritmo_3"/>
     <addaction name="separator"/>
     <addaction name="actionEditar_segmentaci_n"/>
    </widget>
    <addaction name="actionAgregar_referencia_ocular"/>
    <addaction name="separator"/>
    <addaction name="actionFusionar"/>
    <addaction name="actionRectificar"/>
    <addaction name="menuSegmentar"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
    <addaction name="actionManual_de_usuario"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuOCT"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionRehacer">
   <property name="text">
    <string>Rehacer</string>
   </property>
  </action>
  <action name="actionDeshacer">
   <property name="text">
    <string>Deshacer</string>
   </property>
  </action>
  <action name="actionCortar">
   <property name="text">
    <string>Cortar</string>
   </property>
  </action>
  <action name="actionCopiar">
   <property name="text">
    <string>Copiar</string>
   </property>
  </action>
  <action name="actionPegar">
   <property name="text">
    <string>Pegar</string>
   </property>
  </action>
  <action name="actionAgregar_referencia_ocular">
   <property name="text">
    <string>Agregar referencia ocular</string>
   </property>
  </action>
  <action name="actionRectificar">
   <property name="text">
    <string>Rectificar</string>
   </property>
  </action>
  <action name="actionFusionar">
   <property name="text">
    <string>Fusionar</string>
   </property>
  </action>
  <action name="actionAlgoritmo_1">
   <property name="text">
    <string>Algoritmo 1</string>
   </property>
  </action>
  <action name="actionAlgoritmo_2">
   <property name="text">
    <string>Algoritmo 2</string>
   </property>
  </action>
  <action name="actionAlgoritmo_3">
   <property name="text">
    <string>Algoritmo 3</string>
   </property>
  </action>
  <action name="actionEditar_segmentaci_n">
   <property name="text">
    <string>Editar segmentaci√≥n</string>
   </property>
  </action>
  <action name="actionImprimir">
   <property name="text">
    <string>Imprimir</string>
   </property>
  </action>
  <action name="actionImagen">
   <property name="text">
    <string>Imagen</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="actionManual_de_usuario">
   <property name="text">
    <string>Manual de usuario</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
